prometheus/README.md

```diff
@@ -0,0 +1,285 @@
+
+kubectl create -f desired-cert.yaml
+```
+**Note**: Actual namespace you use will be based on design requirements. We are using namespace default in this example as this namespace is available in all PKS deployments.   
+
+**Note**: secretName in the certificate request will be referenced by the ingress controller.
+
+## Cert Manager Validation 
+
+To verify your installation has complete:
+```
+root@cli-vm:~/app# kubectl get pod -n cert-manager
+NAME                                   READY     STATUS      RESTARTS   AGE
+cert-manager-6874795dc8-njsxn          1/1       Running     0          1d
+cert-manager-webhook-ca-sync-vsxqc     0/1       Completed   0          1d
+cert-manager-webhook-d85fcf8cf-nln8v   1/1       Running     0          1d
+root@cli-vm:~/app#
+```
+
+Confirm self-signed and signed cert are loaded in key store:
+
+```
+root@cli-vm:~/app# kubectl get secret | grep kubernetes.io/tls
+ca-key-pair                                        kubernetes.io/tls                     2         1d
+example-com-tls                                    kubernetes.io/tls                     3         1d
+root@cli-vm:~/app#
+
+Retrieve the signed TLS key pair
+
+root@cli-vm:~/app#kubectl get secret example-com-tls -o yaml
+
+```
+
+
+## Install Prometheus
+
+We are going to create the following customization file so our Prometheus deployment will leverage persistent volume for metrics and expose Grafana service to external users using an ingress controller, with SSL termination for added security.  Cert manager will issue and maintain the certificate required by the ingress controller. Core Prometheus services will not be externally accessible unless explicit port forwarding is enabled.
+
+```yaml
+alertmanager:
+  alertmanagerSpec:
+    storage:
+      volumeClaimTemplate:
+        spec:
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: 1Gi
+
+prometheus:
+  prometheusSpec:
+    storage:
+      volumeClaimTemplate:
+        spec:
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: 1Gi
+
+grafana:
+  adminPassword: "VMware1!"
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+    hosts:
+      - grafana.test.example.com
+    tls:
+      - secretName: example-com-tls
+        hosts:
+          - grafana.test.example.com
+  persistence:
+    enabled: true
+    accessModes: ["ReadWriteOnce"]
+    size: 1Gi
+```
+
+**Note**: secretName must match name specificied in the certificate sign request.  See desired-cert.yaml from prior steps.
+
+* Install a nginx ingress controller using helm
+
+    `helm install stable/nginx-ingress --name quickstart`
+    
+* We will deploy Promethus components in the monitoring namespace.  Let's create a monitoring namespace : 
+
+    `kubectl create namespace monitoring`  
+    
+  
+* Install Prometheus Operator using helm chart and reference the custom.yml
```

Comment:
where did the `custom.yml` come from? I don't see that `yml` file here in these directions

Reply:
@xgao3 i'm not connecting the dots properly. where is `custom.yml` coming from?

Reply:
fix the spelling mistake and also added the description for custom.yml

Reply:
i see it now. cool! nice job. Can you squash your commits?